### DISCUSSION TITLE
Parameter selector / option picker menu

### DISCUSSION THREAD

**Discussion by DusanLesan** [Kong/insomnia#7032] [Ideas]
I often use requests where I am just changing one query parameter between few of the possible values. 
For example, I have a parameter named `type` where possible values can be `movie`, `series`, `season`, and  `episode`. I very often need to change between them. 

I wanna have another type of the parameter value input (besides prompt, request, response...), where I would be able to easily switch between the list of set values. When I open parameters section, I want to have current selection easily visible, and when I click on it, I want to have a easy way to select any of the existing entries, add one or remove them. This should work for both query and path parameters. 

Picker could look almost the same as the parameters section itself with the Add/Delete All/Toggle description but without he need to have name row (only value and description are needed). Each row could have its own delete and edit button (clicking on the entry should select it so edit would need to be locked by default (in the worst case single click to select and double to edit (worst because I expect we would need to wait for some time before selecting an entry to see if it is not double-click))).

**Comment by WelingtonMonteiro** [comment]
Up

**Comment by Murillo-Pinheiro-Sisfrete** [comment]
Up
